ohhell.py

```diff
@@ -0,0 +1,93 @@
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import List
```

Comment:
Personally I prefer to write:

```python
from __future__ import annotations
ls: list[int] # now we can do this
```  
The upper case typing annotations are being ~decommissioned~ deprecated soon.